static/images/README.md

```diff
@@ -0,0 +1,19 @@
+Add logo with name `logo.svg`.
```

Comment:
Does this file belongs in this repo? I am not sure

Reply:
Removed in 32433c9.